neovim/plugin/decorators.py

```diff
@@ -83,7 +83,7 @@ def dec(f):
     return dec
 
 
-def autocmd(name, pattern='*', sync=False, eval=None):
+def autocmd(name, group=None, pattern='*', sync=False, eval=None):
```

Comment:
Maybe add `group` as the last argument to avoid a breaking change (if someone called the function without keyword arguments)?